media/js/mozorg/contribute/taskview.js

```diff
@@ -22,20 +22,17 @@
      * https://developer.mozilla.org/en-US/docs/Web/API/Page_Visibility_API
      */
     function getVisibilityStateEventKeyword() {
-        // for browser that support the event unprefixed
-        if (typeof document.hidden !== 'undefined') {
-            return 'visibilitychange';
-        }
+        var evtName = undefined;
 
-        // for Chrome 13 and lower
-        if (typeof document.webkitHidden !== 'undefined') {
-            return 'webkitvisibilitychange';
+        if (typeof document.hidden !== 'undefined') {
+            // for browser that support the event unprefixed
+            evtName = 'visibilitychange';
+        } else if (typeof document.webkitHidden !== 'undefined') {
+            // for Chrome 13 and lower
+            evtName = 'webkitvisibilitychange';
         }
 
-        // for older versions of IE
-        if (typeof document.msHidden !== 'undefined') {
```

Comment:
I tested in my Windows 8 VM and IE 10 reported support for/worked with the standard `visibilitychange` event name. Can I use also shows [full support in IE 10](http://caniuse.com/#feat=pagevisibility).

I see a few blogs (including MDN) reference `document.msHidden`, but I couldn't find any official/MS documentation on it, nor did it work in IE 8 or 9. Maybe it was included in a preview release of IE 10 or something?

I suppose it's not a big deal to keep the check in the code, but as far as I can tell it's not doing anything. What do you think?